deltachat-ios/Controller/ChatListController.swift

```diff
@@ -250,11 +250,11 @@ class ChatListController: UITableViewController {
     private var isInitial = true
     @objc func applicationDidBecomeActive(_ notification: NSNotification) {
         if navigationController?.visibleViewController == self {
-            if !isInitial {
+            if isInitial {
                 isInitial = false
                 startTimer()
-                refreshInBg()
             }
+            refreshInBg()
```

Comment:
i think `refreshInBg` is too much here, no need to reload the chatlist, this is driven by the events already. it should be sufficient to redraw the cells.

(the bug kind of proves that - applicationDidBecomeActive has never done any update - and the chatlist was always okay in my experience - apart of the timestamps :)

Reply:
It's not really affecting the perfomance since the whole thing runs in the background and is even debounced. This is also the same what we currently do in the timer that is started by `startTimer()`, so it's also more consistent right now than calling directly handleChatListUpdate().

Reply:
sure, it does not block ui, but doing an expensive request that costs maybe 2 seconds processor time every 60 seconds will still affect overall performance and resources. and, if the call is not needed, it seems to be worthwhile to avoid it.

if `handleChatListUpdate()` just forces the redraw - what's wrong with calling it? 

@cyBerta  we can also discuss this in an av, i am available today :)

Reply:
After reviewing it, the timer registration logic is just wrong

Reply:
> After reviewing it,I realized the timer registration logic is just wrong

fixed with last commit